api/service.ts

```diff
@@ -62,6 +62,11 @@ export interface Container {
      * as it is running.
      */
     environment?: {[name: string]: pulumi.Input<string>};
+    /**
+     * Number of CPUs for the container to use. Maps to the Docker `--cpus` option - see
+     * https://docs.docker.com/engine/reference/commandline/run.
+     */
+    cpu?: pulumi.Input<number>;
     /**
```

Comment:
I think this is not what the ECS `cpu` option maps to, and I think is also not the commonly used CPU parameter. See 
https://docs.aws.amazon.com/AmazonECS/latest/APIReference//API_ContainerDefinition.html.

Reply:
Will fix today